src/ansys/fluent/core/solver/flobject.py

```diff
@@ -679,56 +679,62 @@ def _get_new_keywords(obj, kwds):
     return newkwds
 
 
-class Command(Base):
-    """Command object."""
+class Action(Base):
+    """Intermediate Base class for Command and Query class."""
 
     def __init__(self, name: str = None, parent=None):
-        """__init__ of Command class."""
+        """__init__ of Action class."""
         super().__init__(name, parent)
         if hasattr(self, "argument_names"):
             for argument in self.argument_names:
                 cls = getattr(self.__class__, argument)
                 self._setattr(argument, cls(None, self))
 
-    def arguments(self) -> Any:
-        """Get the arguments for the command."""
+    def arguments(self, class_name) -> Any:
```

Comment:
No need to pass the `class_name` argument here.  `self.__class__.__name__` should work, and so there is no need for this method in the derived class.

Reply:
Done. Thank you.